### PR TITLE
Login Page added

### PR DESCRIPTION
**Pull request by rojandahal**
-Login Page UI created

### CODE REVIEW

File: src/components/SideBar/BrandNavBar.jsx
Comment (rabin245) [inline src/components/SideBar/BrandNavBar.jsx:1]: navbar should be inside sidebar

File: src/pages/Authentication/Login/Login.jsx
Comment (rabin245) [inline src/pages/Authentication/Login/Login.jsx:1]: refactor reused parts of the ui into its own component

**Review by rabin245 — CHANGES_REQUESTED** [review]
change as requested

File: src/pages/Authentication/Login/Login.jsx
Comment (rojandahal) [reply src/pages/Authentication/Login/Login.jsx:1]: Done

**Pull request merged**